src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -115,7 +114,7 @@ public async Task<HttpResponse> SendRequestAsync(
                     bindingCertificate,
                     customHttpClient,
                     cancellationToken: cancellationToken,
-                    retryCount) // Pass the updated retry count
+                    retryCount++) // increment the retry count
```

Comment:
nit: 
```suggestion
                    retryCount++) // increment and pass the retry count
```

Reply:
I feel that "pass" is already pretty self explanatory since it's used as a parameter here

Reply:
I added this since this is the only parameter that changes between the retry requests.